core/src/main/java/com/novoda/noplayer/AdvertView.java

```diff
@@ -0,0 +1,14 @@
+package com.novoda.noplayer;
+
+public interface AdvertView {
+
+    NoPlayer.AdvertListener getAdvertListener();
+
+    void attach(AdvertInteractionListener advertInteractionListener);
+
+    void detach(AdvertInteractionListener advertInteractionListener);
+
+    interface AdvertInteractionListener {
+        void onAdvertClicked();
+    }
+}
```

Comment:
Should we have comments on this type and it's method? Given that it's a public API

Reply:
Absolutely, I'll add some now!

Reply:
Actually, would you be ok if I create a ticket for this and do a complete sweep of the library? Some other places, not related to this have been missed.

Reply:
https://github.com/novoda/no-player/issues/219